lib/localparticipant.js

```diff
@@ -223,7 +223,7 @@ LocalParticipant.prototype._getOrCreateLocalTrackPublication = function _getOrCr
       var error = trackSignaling.error;
       if (error) {
         trackSignaling.removeListener('updated', updated);
-        log.warn('Failed to publish the ' + util.trackClass(localTrack, true) + ':', error);
+        log.warn('Failed to publish the ' + util.trackClass(localTrack, true) + ':' + error.message);
```

Comment:
@markandrus `':' + error.message);`: now that we've replaced the `,` with a `+`, can we add a space after `:`?